WiserTaskScheduler/WiserTaskScheduler/Modules/Slack/modules/SlackSettings.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace WiserTaskScheduler.Modules.Slack.modules;
+
+public class SlackSettings
+{ 
+  /// <summary>
+  /// GET SLACK CHANNEL NAME
+  /// </summary>
+  public  string SlackChannel { get; set; }
```

Comment:
Graag 4 spaties inspringen ipv 2, zoals we op alle andere plekken ook doen. En commentaar niet in hoofdletters, maar gewoon normale zin van maken die ook eindigt met een punt.